### DISCUSSION TITLE
How to use a mirror to upgrade, if outgoing connections are blocked?

### DISCUSSION THREAD

**Discussion by beyondtop** [datenstrom/community#792] [Ask a question] [answered]
"Can't connect to server 'https://github.com'!"
How to use mirror 'GitHub' to upgrade? Thank you

**Comment by IngoBelka** [comment]
May be this helps you: [#763](https://github.com/datenstrom/yellow/discussions/763#discussioncomment-2338957)

**Reply by beyondtop** [reply]
https://github.com/ URLs are  blocked.The hosting provider confirmed that outgoing connections were blocked as default. They can't switched them on.So...

**Comment by schulle4u** [comment, marked as answer]
Do you have access to GitLab? Maybe we could create an automatic mirror for our repositories there. 

Not tested, but as a temporary solution you can try to download the yellow-extensions repository from a machine with GitHub access, then upload the whole folder somewhere else to a web-accessible directory. Change `UpdateExtensionUrl` in your `system/extensions/yellow-system.ini` and point it to the new directory (e.g `https://example.com/yellow-extensions`). Now you should be able to update your website.

**Reply by beyondtop** [reply]
Thank you.
